src/app/components/game-components/game-component/game-component.scss

```diff
@@ -21,6 +21,9 @@ $victory: yellow;
     stroke: $base-stroke;
     stroke-width: 8;
     fill: $background-fill;
+    stroke-linecap: butt;
+    stroke-linejoin: round;
```

Comment:
round butt ? what's that pornography, sir ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linejoin et https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linecap !